src/test/java/com/revature/services/ProductServiceTest.java

```diff
@@ -110,6 +123,18 @@ void testDelete() {
 		this.pServ.delete(this.dummyProduct.getId());
 		verify(this.mockProductRepo, times(1)).deleteById(this.dummyProduct.getId());
 	}
+	
+	@Test
+	void testDelete_Failure_ProductNotFound() {
+		int id = 404;
+		given(this.mockProductRepo.findById(id)).willReturn(Optional.empty());
+		try {
+			this.pServ.delete(id);
+			fail("Expected an exception to be thrown");
```

Comment:
You should specify which exception type is expected to be thrown. In this case, it's `ProductNotFoundException`